src/osx/cocoa/utils_base.mm

```diff
@@ -25,7 +25,7 @@
 // our OS version is the same in non GUI and GUI cases
 wxOperatingSystemId wxGetOsVersion(int *majorVsn, int *minorVsn)
 {
-#if MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_X_VERSION_10_10
+#if MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_X_VERSION_10_10 || defined(__WXOSX_IPHONE__)
```

Comment:
It looks like I need to test for `__IPHONE_8_0` and use `--with-macosx-version-min=7.0` and **also** avoid testing for `MAC_OS_X_VERSION_MAX_ALLOWED` which is still defined even for iOS build. Anyhow, I'll fix this.